models/lung_nodule_ct_detection/configs/train.json

```diff
@@ -0,0 +1,450 @@
+{
+    "imports": [
+        "$import glob",
+        "$import os"
+    ],
+    "bundle_root": "./",
+    "ckpt_dir": "$@bundle_root + '/models'",
+    "output_dir": "$@bundle_root + '/eval'",
+    "data_list_file_path": "$@bundle_root + '/annotation/dataset_fold0.json'",
+    "data_file_base_dir": "/home/canz/Projects/datasets/LUNA16/93176/Images_resample",
+    "train_datalist": "$monai.data.load_decathlon_datalist(@data_list_file_path, is_segmentation=True, data_list_key='training', base_dir=@data_file_base_dir)",
+    "device": "$torch.device('cuda:0' if torch.cuda.is_available() else 'cpu')",
+    "epochs": 300,
+    "num_interval_per_valid": 1,
+    "learning_rate": 0.01,
+    "amp": true,
+    "batch_size": 3,
+    "patch_size": [
+        192,
+        192,
+        80
+    ],
+    "val_patch_size": [
+        512,
+        512,
+        208
+    ],
+    "anchor_generator": {
+        "_target_": "monai.apps.detection.utils.anchor_utils.AnchorGeneratorWithAnchorShape",
+        "feature_map_scales": [
+            1,
+            2,
+            4
+        ],
+        "base_anchor_shapes": [
+            [
+                6,
+                8,
+                4
+            ],
+            [
+                8,
+                6,
+                5
+            ],
+            [
+                10,
+                10,
+                6
+            ]
+        ]
+    },
+    "backbone": "$monai.networks.nets.resnet.resnet50(spatial_dims=3,n_input_channels=1,conv1_t_stride=[2,2,1],conv1_t_size=[7,7,7])",
+    "feature_extractor": "$monai.apps.detection.networks.retinanet_network.resnet_fpn_feature_extractor(@backbone,3,False,[1,2],None)",
```

Comment:
If these 2 components are class instances, I think we should define with `_target_` args?
@yiheng-wang-nv What do you think?

Thanks.

Reply:
Thanks Nic. They are not class instances. They are functions.